themes/default/content/product/_index.md

```diff
@@ -4,13 +4,18 @@ meta_desc: Learn how Pulumi's products enable your team to get code to any cloud
 type: page
 layout: product
 
-heading: Pulumi Cloud Engineering Platform
+heading: Pulumi Overview
```

Comment:
very supportive of this change, but i believe that marketing had previously advocated against this as they were trying to build up the cloud engineering connection.

Reply:
I know :-)  I still explained what Cloud Engineering is on the page, but the current experience just doesn't make much sense IMHO -- especially now that we're leaning heavily on the IaC messaging.